src/containers/my-resources/my-resources-table/MyResourcesTable.tsx

```diff
@@ -88,7 +90,7 @@ const MyResourcesTable = <T extends TableItem>({
 
   const rowActions: TableRowAction[] = [
     {
-      label: t('common.edit'),
+      label: !rename ? t('common.edit') : t('common.rename'),
```

Comment:
As rename is used only for Categories, you can check it by **ResourcesTabsEnum.Categories**

label: resource === ResourcesTabsEnum.Categories ? t('common.rename') : t('common.edit')

Reply:
yeah thats better